tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -130,8 +106,8 @@ def test_get_replays_require_timely_initial_sequence(self):
         replay1_timestamp0 = datetime.datetime.now() - datetime.timedelta(days=365)
         replay1_timestamp1 = datetime.datetime.now() - datetime.timedelta(seconds=10)
 
```

Comment:
this test wasn't actually doing what it said on the tin -- it was reliant on the duration check as since the first replay  is dropped as its too old, a 0 duration replay remained and was excluded by the old filter. 

I've added segment id 0 / 1 here which fixes the behavior of the test.